docs/en/maintain/backup.md

```diff
@@ -0,0 +1,202 @@
+# Backup and restore
+
+## High Availability Description
+
+* The nameserver cannot be fully hung up. Failover and recovery cannot be done if all hangs up
```

Comment:
The nameserver nodes cannot all fail. Failover and recovery cannot be done if all fail.